frontend/src/dashboard/dashboard.html

```diff
@@ -1,8 +1,19 @@
 <div class="dashboard">
-  <div v-if="!dashboard">
-    <div v-for="dashboard in dashboards" :key="dashboard._id" @click="$router.push('dashboards?dashboardId=' + dashboard._id)">
-      {{dashboard.name}}, ID: {{ dashboard._id }}
+  <div v-show="dashboard">
+    <div>
+      <h2>{{name}}</h2>
     </div>
+    <div>
+      <pre>{{code}}</pre>
+      <button @click="toggleEditor" style="color: black;margin-right: 0.5em">Edit</button>
+    </div>
+    <div v-show="showEditor">
+      <textarea class="border border-gray-200 p-2 h-[300px] w-full" ref="codeEditor"></textarea>
```

Comment:
Please make this a separate `edit-dashboard` component in `frontend/src/dashboard/edit-dashboard`. That will make it much easier to handle weirdnesses with mounting CodeMirror. Because right now, CodeMirror doesn't show up until you click on the textarea, likely because it isn't visible by default. With a separate component and a `v-if`, you can just use the `edit-dashboard` component's `mounted()` hook